stable/g2/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
This is an outdated version of the file with wrong truncation. Update it from this example: https://github.com/kubernetes/helm/blob/master/docs/examples/nginx/templates/_helpers.tpl